test/topojson/test.ml

```diff
@@ -98,24 +109,28 @@ let ezjsonm =
     (fun ppf t -> Fmt.pf ppf "%s" (Ezjsonm.value_to_string t))
     ezjsonm_equal
 
+(* type pp_set_formatter_out_functions = {
+     foreign_member : Topojson.Geometry.foreign_members -> unit;
+
+   } *)
 let main () =
   let s = read_file "./test_cases/files/exemplar.json" in
   let json = Ezjsonm.value_from_string s in
+
   let pp_ezjsonm ppf json = Fmt.pf ppf "%s" (Ezjsonm.value_to_string json) in
-  let pp_foreign_member ppf (v : (string * Ezjsonm.value) list) =
+  let pp_foreign_member ppf (v : (string * get_foreign_members_in_point) list) =
     Fmt.pf ppf "%a" Fmt.(list (pair string pp_ezjsonm)) v
   in
-
   let foreign_members = Alcotest.testable pp_foreign_member Stdlib.( = ) in
-  let expected_foreign_members = [ ("arcs", json) ] in
```

Comment:
You will have to write the expected foreign members by hand by looking at the JSON file. What we need here is

```ocaml
let expected_foreign_members = [ ("arcs", `A [ `Float 0. ]) ] in
```